remote-api/src/main/java/org/alfresco/rest/api/impl/RenditionsImpl.java

```diff
@@ -582,28 +587,32 @@ protected NodeRef getRenditionByName(NodeRef nodeRef, String renditionId, Parame
                 throw new InvalidArgumentException("renditionId can't be null or empty.");
             }
 
-            ChildAssociationRef nodeRefRendition = renditionService2.getRenditionByName(nodeRef, renditionId);
-            if (nodeRefRendition != null)
-            {
-                ContentData contentData = getContentData(nodeRefRendition.getChildRef(), false);
-                if (contentData != null)
-                {
-                    return tenantService.getName(nodeRef, nodeRefRendition.getChildRef());
-                }
-            }
+            RenditionContentData renditionContentData = renditionService2.getRenditionByName(nodeRef, renditionId);
+            return renditionContentData;
+            //TODO - investigate if we need tenant service?
```

Comment:
Move TODO ? 

As long as NodeRef is not exposed then multi-tenancy (MT) should be handled at a lower layer, eg. where we enable optional switching from RenditionService2 to RenditionService3 (or back again).